smrf/distribute/wind/wind.py

```diff
@@ -81,22 +78,34 @@ def __init__(self, config):
         self.smrf_config = config
         self.getConfig(config['wind'])
 
-        if self.config['wind_model'] == 'interp':
+        if self.check_wind_model_type('interp'):
             # Straight interpolation of the wind
             self.wind_model = self
             self.wind_model.flatwind = None
             self.wind_model.cellmaxus = None
             self.wind_model.dir_round_cell = None
 
-        elif self.config['wind_model'] == 'wind_ninja':
+        elif self.check_wind_model_type('wind_ninja'):
```

Comment:
For future improvements, I would declare a constants in each class (`WindNinja` and `Windstral`) and then pass this as an argument. There are several other comparisons in this  class against this string and it would be nice to have one central definition of that.